typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -65,29 +65,14 @@ const nativeBridges = {
   },
 };
 
-type L2Chain =
-  | Chains.optimism
-  | Chains.optimismgoerli
-  | Chains.arbitrum
-  | Chains.arbitrumgoerli
-  | Chains.base;
-
-const L2Chains: ChainName[] = [
-  Chains.optimism,
-  Chains.optimismgoerli,
-  Chains.arbitrum,
-  Chains.arbitrumgoerli,
-  Chains.base,
-  Chains.polygonzkevmtestnet,
-];
+type L2Chain = Chains.optimism | Chains.arbitrum | Chains.base;
+
+const L2Chains: ChainName[] = [Chains.optimism, Chains.arbitrum, Chains.base];
 
 const L2ToL1: ChainMap<ChainName> = {
-  optimismgoerli: 'goerli',
-  arbitrumgoerli: 'goerli',
   optimism: 'ethereum',
   arbitrum: 'ethereum',
   base: 'ethereum',
-  polygonzkevmtestnet: 'goerli',
```

Comment:
as mentioned in description, we will no longer bridge L1->L2 for polygonzkevmtestnet. This means we have to manually top up funder there from now on

Reply:
cc @nambrot

Reply:
loooks like that one will also get removed, so tbh i'm fine with removing polygonzkevm testnet as well https://polygon.technology/blog/polygon-pos-and-polygon-zkevm-new-testnets-for-polygon-protocols

Reply:
nice, I'll take that down too then